### DISCUSSION TITLE
Copilot is taking VSCODE slow to save files and load references on Python projects

### DISCUSSION THREAD

**Discussion by israelmelicio** [community/community#9012] [Copilot Conversations] [answered]
The last 3 versions of Copilot is causing an estrange behavior on VSCODE for Python projects (at the time of this discussion the last version is 1.7.4406)

It's take a long time to load references, such as in my imports (take about 2 minutes in my Mac Pro M1 13") , and to save a file, its  causing a waiting message like thatL `Saving 'some_file.py': Getting code actions from ''Jupyter', 'GitHub Pull Requests and Issues'' (configure).` The extensions Jupter and GitHub Pull Requests and Issues are replaced for any other, if I disable then.

However when I downgrade to version 1.7.4016, these behavior does not happen.

Does anybody is facing this problem?

P.S. My project has about 12k files.

**Comment by luizeboli** [comment]
Related to https://github.com/github/feedback/discussions/8861

**Comment by hsenag** [comment, marked as answer]
Hi, sorry about this. There were some performance bugs we believe we have now fixed. Please try the latest version. As @luizeboli says, #8861 has the full details.

**Reply by israelmelicio** [reply]
It's working now. Many thanks to @hsenag! And thank you by related topic warning @luizeboli!
